src/app/containers/StoryPromo/index.jsx

```diff
@@ -109,6 +109,7 @@ const StoryPromoContainer = ({
   lazyLoadImage,
   dir,
   displayImage,
+  displaySummary,
```

Comment:
Prior to adding this prop, the summary was only displayed when it exists (truthy), and `displayImage` is `true`. Now, with the prop, it's explicit. 

This change is necessary because the Top Stories contain no images, and yet, we need to show the summary.